### PR TITLE
update template

### PR DESCRIPTION
**Pull request by dariusptrs**
for generating a CI status badge

### CODE REVIEW

**Comment by hofbi** [comment]
I would not start adding a dummy GH action to every repo just for getting a badge. We can roll this out over time whenever someone wants to change something in the repective repo. Since you already started touching this one, we could switch the travis CI to our proper GH action.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 build/
 
-Systemtheorie.pdf
+/*.pdf
```
Comment (hofbi) [inline .gitignore]: This can cause images to be ignored. We must be explicit here

Comment (dariusptrs) [reply .gitignore]: I think the slash causes it to only ignore files in the root directory (excluding subdirectories)

Comment (hofbi) [reply .gitignore]: And if someone add a PDF into the root dir?

Comment (dariusptrs) [reply .gitignore]: Yes, any pdf files directly in the root directory would be ignored. PDF's in other directories such as `img` will not be ignored.

Comment (hofbi) [reply .gitignore]: My question is if this is the expected behavior or if we should stick to explicitly ignoring this single file.

Comment (dariusptrs) [reply .gitignore]: I think it is a good practice to have all image files in a separate directory. So for me this is expected behavior, but I can also change it back to only ignore the single file.

Comment (hofbi) [reply .gitignore]:
```suggestion
*.pdf
!img/*.pdf
```

How about this?

Comment (dariusptrs) [reply .gitignore]: Okay

**Review by hofbi — APPROVED** [review]

**Pull request merged**